server/routes/viewPatientSearch/index.ts

```diff
@@ -3,10 +3,10 @@ import asyncMiddleware from '../../middleware/asyncMiddleware'
 
 import PatientSearchRoutes from './patientSearch'
 
-export default function viewPatientRoutes(): Router {
+export default function viewPatientRoutes(searchResultsPath: string): Router {
```

Comment:
If we are going to share this, then i think it is best to use a more generic name and put it in a generic folder/directory/package. How about `restrictedPatientSearch` and `restrictedPatientSearchRoutes`?